From 7099c4d8c96cc06bbd13043c7e43d4330c1e89b2 Mon Sep 17 00:00:00 2001
From: Matt Ranostay <matt.ranostay@intel.com>
Date: Tue, 5 May 2015 16:43:37 -0700
Subject: [PATCH 393/462] atomisp: fix atomisp_fops.c compile error

Issue was caused due to gcc toolchain update

Change-Id: Ief866019baeb9590a306d329a4734b535d1f9fcd
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-36364
Signed-off-by: Matt Ranostay <matt.ranostay@intel.com>
Signed-off-by: Ding Yunliang <yunliang.ding@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/55953
Approver: Bingbu Cao <bingbu.cao@intel.com>
Reviewed-by: Bingbu Cao <bingbu.cao@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: IMIN (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_fops.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_fops.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_fops.c
index 6a5265a..7423122 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_fops.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_fops.c
@@ -767,10 +767,10 @@ static int atomisp_open(struct file *file)
 {
 	struct video_device *vdev = video_devdata(file);
 	struct atomisp_device *isp = video_get_drvdata(vdev);
-	struct atomisp_video_pipe *pipe;
-	struct atomisp_acc_pipe *acc_pipe;
+	struct atomisp_video_pipe *pipe = NULL;
+	struct atomisp_acc_pipe *acc_pipe = NULL;
 	struct atomisp_sub_device *asd;
-	bool acc_node;
+	bool acc_node = false;
 	int ret;
 
 	dev_dbg(isp->dev, "open device %s\n", vdev->name);
-- 
1.9.1

